plugins/inputs/jolokia/jolokia.go

```diff
@@ -14,7 +14,16 @@ import (
 	"github.com/influxdata/telegraf/plugins/inputs"
 )
 
-type Server struct {
+type JolokiaServer struct {
+	Name     string
+	Host     string
+	Username string
+	Password string
+	Port     string
+	Https    bool
+}
```

Comment:
My $0.02: I think instead of a half-dozen config params, the plugin should just accept a URL.